opentelemetry-sdk/src/metrics/internal/aggregate.rs

```diff
@@ -113,6 +113,7 @@ impl<T: Number<T>> AggregateBuilder<T> {
         // a last-value aggregate.
         let lv_filter = Arc::new(LastValue::new());
         let lv_agg = Arc::clone(&lv_filter);
+        let t = self.temporality;
```

Comment:
nit - should we update the function comment, as the temporality is no longer ignored now?

Reply:
Thanks! Updated.